### PR TITLE
CSS Grid & Sound Actions

### PR DESCRIPTION
**Pull request by lmangani**
Opening for review/approval and merge. This PR introduces a few changes:

- drop gridstack for our own autopilot CSS driven grid (light) and autoscaling
- speaker meter and event emitter (i'm talking)
- grid shuffle based on active speaker event (move to 1st block)
- sound effects on joining room
- additional popups on user interaction (mute, etc)
- video poaster with a light gif (mostly for determining the dead object in debugging)

NOTE: drag & resize of widgets will be reintroduced later

### CODE REVIEW

**Comment by lmangani** [comment]
@QVDev are we ok with this PR?

File: src/assets/js/rtc.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
  */
 import h from "./helpers.js";
 import EventEmitter from "./emitter.js";
 import Presence from "./presence.js";
 import MetaData from "./metadata.js";
+
 var TIMEGAP = 6000;
 var allUsers = [];
-var enableHacks = false;
+var enableHacks = true;
```
Comment (QVDev) [inline src/assets/js/rtc.js:14]: We want to have this on true?

Comment (lmangani) [reply src/assets/js/rtc.js:14]: good point. the only effect is this allows clients without webcam/mic to join as receive-only. We can disable it.

Comment (lmangani) [reply src/assets/js/rtc.js]: done!

**Pull request merged**